clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GoogleOperationPoller.groovy

```diff
@@ -105,7 +105,8 @@ class GoogleOperationPoller {
 
       totalTimePollingSeconds += pollInterval
 
-      Operation operation = getOperation()
+      def retry = new SafeRetry<Operation>()
+      Operation operation = retry.doRetry(getOperation, "wait", "operation", null, null, [], [])
```

Comment:
Hm, looks strange to call a static method via an instance reference.
Why not drop the `static` from the `doRetry()` method?